elements/rh-navigation-secondary/docs/10-style.md

```diff
@@ -100,7 +100,7 @@
 
 
   {% alert state="info", title="Helpful Tip" %}
-    Slot 3 usually includes a call to action. To see call to action styles, visit the <a href="../call-to-action">Call to Action</a> documentation.
+    Slot 3 usually includes a Call to action. To see Call to action styles, visit the <a href="../call-to-action">Call to action</a> page.
```

Comment:
Would we want to do this as `[Call to action](/elements/call-to-action)` instead?